Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "AppleAccelerate"
+uuid = "13e28ba4-7ad8-5781-acae-3021b1ed3924"
+version = "0.3.0"
+
+[compat]
+julia = "≥ 1.0.0"
```

Comment:
In the future, it is probably worth just writing `julia = 1` (which declares support for all julia 1.x versions). Declaring compat with julia 2.0 may not make sense.